cogs/stats.py

```diff
@@ -0,0 +1,29 @@
+import discord
+from discord.ext import commands
+from random import choice, randrange
+
+from utils.messages import embedded_stats
+from utils.dummy_data import get_dummy_data
+
+
+class Stats(commands.Cog):
```

Comment:
What are you thoughts on changing this (back?) to `Commands` and the file name to `Commands.py`? Reason: `Commands.py` more accurately reflects the contents of the file.

Reply:
The reason I have it as `stats.py` is because I wanted to isolate commands based on what they are used for.  This lets us future proof the file names (**ex**. we add commands to make channels and we put it in something like `administrative.py`).